docs/CasbinRBACAndRBAC96.mdx

```diff
@@ -1,8 +1,11 @@
 ---
 id: rbac-96
-title: Casbin RBAC v.s. RBAC96
+title: Casbin RBAC And RBAC96
```

Comment:
`v.s.` seems to be a wrong abbreviation, the correct abbreviation should be `vs` or `vs.` (see https://www.zhihu.com/question/24199500/answer/26997500).
It also seems proper to change it to `And`.

Reply:
@Selflocking `vs.` is OK